shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -35,35 +63,94 @@
     private static final float SCROLL_POSITION_CAP_FOR_INFINITY = 70000.0f;
     private static final int ROOT_NODE_ID = 0;
 
-    private final FlutterView owner;
+    /// Value is derived from ACTION_TYPE_MASK in AccessibilityNodeInfo.java
+    private static int FIRST_RESOURCE_ID = 267386881;
+
+    // Real Android View, which internally holds a Flutter UI.
+    @NonNull
+    private final View rootAccessibilityView;
+
+    // Direct interface between Flutter's Android embedding and the Flutter framework.
+    @NonNull
+    private final FlutterJNI flutterJNI;
+
+    // The accessibility communication API between Flutter's Android embedding and
+    // the Flutter framework.
+    @NonNull
     private final AccessibilityChannel accessibilityChannel;
+
+    // The top-level Android View within the containing Window.
+    // TODO(mattcarroll): why do we need this? and why are we caching it?
+    @NonNull
```

Comment:
This is to calculate an additional offset when the screen is rotated to reverse landscape.

Reply:
Do you remember where the offset is applied, and why it's needed? Also, what if we're not displaying Flutter fullscreen?  Does this still make any sense?

Reply:
It should be applied below where we compute the rects. See https://github.com/flutter/flutter/issues/19967 for more context and the linked PR that fixed it.

I'm not sure if it would be required for splitscreen, since splitscreen doesn't rotate (right?)